go/libraries/utils/filesys/inmemfs.go

```diff
@@ -108,8 +120,10 @@ func NewInMemFS(dirs []string, files map[string][]byte, cwd string) *InMemFS {
 				panic("Initializing InMemFS with invalid data.")
 			}
 
-			newFile := &memFile{path, val, targetDir}
+			now := time.Now()
```

Comment:
If we're going to the trouble of having an in-memory file system, we probably want to have it use a fake clock for this kind of thing. Maybe in a second pass? That would make it easier to write repeatable tests for mtime.

Reply:
Put in something quick and dirty.